src/VisualStudio/Core/Def/ServicesVisualStudio.csproj

```diff
@@ -19,7 +19,25 @@
     <DeployExtension>false</DeployExtension>
     <UseCodebase>true</UseCodebase>
     <CopyNuGetImplementations>false</CopyNuGetImplementations>
+    <ResolveAssemblyWarnOrErrorOnTargetArchitectureMismatch>None</ResolveAssemblyWarnOrErrorOnTargetArchitectureMismatch>
   </PropertyGroup>
+  <!-- We need to mark our NuGet.VisualStudio package reference as EmbedInteropTypes.  However,
+       as this package comes from project.json there is no way to do that in the json file.
+       To deal with that, we write this target which finds that added package, removes it from
+       the references list, then adds it back in again with the 'EmbedInteropTypes' flag properly
+       set. -->
+  <Target Name="MarkNuGetVisualStudioPackageWithEmbedInteropType" AfterTargets="ResolveNuGetPackageAssets">
```

Comment:
Still I don't think this is quite right.  We only do this in one .csproj, but multiple project.json's are now pulling in this reference because of transitivity.  Maybe we're getting lucky and no references are being burned in for the other projects because we don't directly reference it there, but that seems fragile.